### ISSUE TITLE
Map preview doesn't load service (HTTPS vs HTTP mismatch problem?)

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#126]
When creating a service using the deployed version at Section917 (http://section917.canadacentral.cloudapp.azure.com/respect/index-ca-en.html#/en/editor), previewed maps are served from the ArcGIS service over HTTP instead of the HTTPS URL specified in the map config. This results in an error in the preview, when the published/saved map works just fine.

**Comment by RyanCoulsonCA** [comment]
Yeah, I'm pretty sure this is happening due to an HTTP/HTTPS mismatch. The error being returned by the service is "Cannot proxy HTTP request, ArcGIS Server has been configured to accept only HTTPS requests. Please configure your Web Server to use SSL/HTTPS."

Currently the editor needs to be hosted on HTTP because the Express server can't run on HTTPS unless it is configured to do so (which requires a key and a certificate, not exactly familiar with how to generate these at the moment). If anyone knows how to do this let me know. Alternatively, if it's possible to re-configure the ArcGIS server to allow HTTP requests that may fix it too.

**Comment by RyanCoulsonCA** [comment]
@mohsin-r brought [this PR](https://github.com/fgpv-vpgf/fgpv-vpgf/pull/4088) to my attention from RAMP3 which will most likely fix this issue. Looks like there's some logic in the RAMP3 code that we're using that tries to match protocols.
